tests/test_py_counter.py

```diff
@@ -48,3 +56,13 @@ def test_count_classes_with_docstrings():
     """Test case for counting the number of Classes that do not have docstrings."""
     expected_classes_dst_2 = 1
     assert test_pycount.count_classes_with_docstring() == expected_classes_dst_2
+
+def test_find_parameters():
+    """Test case for counting the parameters"""
+    expected_parameters = 0
+    assert test_pycount.count_function_parameters() == expected_parameters
+
+def test_count_assignment_statements():
+    """Test case for counting assignement statements."""
+    expected_assignment_statements = 2
+    assert test_pycount.count_assignment_statements() == expected_assignment_statements

```

Comment:
This test should be updated once `count_assignment_statements` has been updated to count `AugAssign` nodes as well.

Reply:
@solisa986